cluster/gce/util.sh

```diff
@@ -2790,139 +2792,55 @@ function create-proxy-vm() {
   return 1
 }
 
-function setup-proxy() {
-  ssh-to-node ${PROXY_NAME} "sudo sed -i '\$afs.file-max = 1000000' /etc/sysctl.conf"
-  ssh-to-node ${PROXY_NAME} "sudo sysctl -p"
-  ssh-to-node ${PROXY_NAME} "sudo sed -i '\$a*       hard    nofile          1000000' /etc/security/limits.conf"
-  ssh-to-node ${PROXY_NAME} "sudo sed -i '\$a*       soft    nofile          1000000' /etc/security/limits.conf"
-  ssh-to-node ${PROXY_NAME} "sudo apt update -y"
-  ssh-to-node ${PROXY_NAME} "sudo apt install -y nginx"
-  ssh-to-node ${PROXY_NAME} "sudo rm -f /etc/nginx/nginx.conf"
+function init-proxy-cfg() {
 
-  RESOURCE_MASTER_IP=${1}
+  if [[ "${KUBERNETES_SCALEOUT_PROXY_APP}" == "haproxy" ]]; then 
```

Comment:
FYI. please checkout the logic in the PR849 for this, for single TP, multiple TP and single RP URL replacement in the conf file